### ISSUE TITLE
Device window keeps opening randomly in Reaper with Novation 49SL mk III

### ISSUE THREAD

**Issue by mydogspies** [git-moss/DrivenByMoss#289]
Seems that the device window, no matter if device has been chosen or not, keeps opening randomly, for example when adding an empty clip, it is added but device window opens on top too.

Also, for some reason , the grid scroll buttons do not work. Even if I add a third clip, all I see is two rows of buttons and the arrow buttons up and down are dimmed. 

Running Reaper, version 6.53 2ith SL49, mk III with firmware 1.4 on a fully updated Windows 10 Pro system. Only other active USB device  is an Yamaha MG10XU mixer using its asio drivers.

Tried to find similar topic elsewhere but Google gives no results.

**Comment by git-moss** [comment]
Is there one case where you can repeat this behaviour and give some steps to reproduce? Is this randomly opening caused by an (unrelated) action or without you interacting?

**Comment by mydogspies** [comment]
Yes, following these steps, it's reproducable every single time, also after computer shutdown and restart, and shutdown and new start of the Novation 49SL;

1) Repower everything.
2) Start the SL
3) Open Reaper in default empty template
4) Go INConnect on the SL and choose MASTER track.
5) Add empty instrument
6) Add clip -> clip added but device window opens and clip starts playing
7) Add any number of clips after the first one, only 2 clip rows show on SL grid - arrow buttons greyed out.

The SL has latest firmware 1.4 version 386, Reaper is a clean install without addons and the Novation USB drivers are the latest.

Video of the bug; 
[sl-reaper-drivebymoss-issue.zip](https://github.com/git-moss/DrivenByMoss/files/8411848/sl-reaper-drivebymoss-issue.zip)

**Comment by git-moss** [comment]
Thanks! Got it. It strangely opened the dialog on clip selection. Will be fixed in the next update!
